<internal_error>
<sub_system>ERR</sub_system>
<callstack>
	0x7efd4317bbe6: ccl_err + 0x7be6 (_ZN15ERR_STACKWALKER15get_stack_traceEPPKviiPv + 0xd8)
	0x7efd4317ee95: ccl_err + 0xae95 (_Z14err_terminatorv + 0x5a)
	0x7efd5c0ebae6: jtag_client + 0x82ae6 (_ZN10__cxxabiv111__terminateEPFvvE + 0x6)
	0x7efd5c0ebb31: jtag_client + 0x82b31
	0x7efd5c0ebc86: jtag_client + 0x82c86 (__cxa_rethrow + 0x46)
	0x7efd577b5034: QtCore.so.4 + 0x1b5034 (_ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE + 0x2f4)
	0x7efd577b9c84: QtCore.so.4 + 0x1b9c84 (_ZN16QCoreApplication4execEv + 0xb4)
	   0x401eab: quartus + 0x1eab (_Z8qgq_mainiPPKc + 0x7b)
	0x7efd43a88e30: ccl_msg + 0x3ee30 (_Z15msg_main_threadPv + 0x10)
	0x7efd43145acc: ccl_thr + 0x5acc (thr_final_wrapper + 0xc)
	0x7efd43a88eef: ccl_msg + 0x3eeef (_Z18msg_thread_wrapperPFPvS_ES_ + 0x62)
	0x7efd431adf9c: ccl_mem + 0x9f9c (_Z18mem_thread_wrapperPFPvS_ES_ + 0x5c)
	0x7efd4317cb39: ccl_err + 0x8b39 (_Z18err_thread_wrapperPFPvS_ES_ + 0x27)
	0x7efd43145b0f: ccl_thr + 0x5b0f (thr_thread_wrapper + 0x15)
	0x7efd43a8aea1: ccl_msg + 0x40ea1 (_Z12msg_exe_mainiPPKcPFiiS1_E + 0xb2)
	   0x401f91: quartus + 0x1f91 (main + 0x26)
	0x7efd39a01d90: c.so.6 + 0x29d90
	0x7efd39a01e40: c.so.6 + 0x29e40 (__libc_start_main + 0x80)
	   0x401d39: quartus + 0x1d39
</callstack>
<error>*** Fatal Error: Unhandled Exception</error>
<date>Fri Jun  2 11:42:42 2023
</date>
<version>Version 20.1.0 Build 711 06/05/2020 SJ Standard Edition</version>
</internal_error>